cloudify_cli/commands/logs.py

```diff
@@ -79,42 +81,54 @@ def download(output_path, all_nodes, logger):
         if not env.profile.cluster:
```

Comment:
is this cond still good enough ?

Reply:
Yes, since if it is an AIO Manager, the cluster attribute would be an empty dictionary.